connector-model/src/main/java/org/interledger/connector/settings/IlpOverHttpConnectionSettings.java

```
@@ -43,31 +43,31 @@ default long keepAliveSeconds() {
    */
   @Value.Default
   default long connectTimeoutMillis() {
-    return 1000;
+    return 10000;
```

10s seems really long just establish a connection. Seems like 1s was plenty and better to fail fast.

---

Agree; reverted.